js/tls.js

```diff
@@ -3377,6 +3377,15 @@ tls.queue = function(c, record) {
     return;
   }
 
+  if (record.fragment.length() == 0) {
```

Comment:
Minor style nit, in this lib we don't put a space between "if" and parenthesis: `if (` => `if(`. We also always compare using triple equals.